ShotgunImportExport/README.md

```diff
@@ -0,0 +1,20 @@
+Shotgun CSV Import Export
+==========================
+
+This is a simplified API example of how to create a timeline in Unity using a shot sequence from Shotgun.
+
+How to use
+--------
+* Create a new project
+* Import the [package](Bin/ShotgunImportExport.unitypackage) into your project
+* Then through the menu `Window > General > Film/TV toolbox > Shotgun > Import CSV` import the example csv file
+    * Alternatively, change the `k_pathToFile` and `k_importFileName` fields to the file you want to use
+* Make edits to the timeline asset
+* Then through the menu `Window > General > Film/TV toolbox > Shotgun > Import CSV` export to Shotgun-compatible tab-separated values. This will create a file name `export_from_unity.csv` in the same directory as the imported CSV file
```

Comment:
Sorry to do this to you, would you mind changed the menu entries to Window/Film-TV toolbox/Shotgun/Import CSV

Reply:
I don't mind